### PR TITLE
Release - 1.38.0

### PR DESCRIPTION
**Pull request by tgandrews**
### What
Release 1.38.0
Contains:
- [x] - [Static page fixes](https://trello.com/c/hFNnHPkd/464-static-page-fixes-1)
- [X] - Remove auto submit form @justinpjose 
- [X] - [Main navigation](https://trello.com/c/mWbS6Z7x/405-a-menu-navigation-task-1-1)
- [x] - [Data page audit](https://trello.com/c/Kwc4IEJj/460-manually-audit-data-pages-1)
- [x] - [Search page audit](https://trello.com/c/rBQhjlZi/458-manually-audit-search-pages-1)
- [x] - [Misc page audit](https://trello.com/c/iyJLTgYI/461-manually-audit-misc-pages-1)
- [x] - [Non descriptive labelling](https://trello.com/c/VOcmkoGh/381-aa-non-descriptive-labelling-2-task-3-05)
- [x] - [iFrame title](https://trello.com/c/PSf1Up6h/350-a-iframe-missing-title-attributes-task-3-05) 

### How to review
That this only contains the features above

### Who can review
Anyone but me

### CODE REVIEW

**Review by dandwal — COMMENTED** [review]
Everything looks ok, spoke to @justinpjose the task https://trello.com/c/GUBBLQRi/4862-add-sort-by-drop-down-15d included the auto-submit removal. Not yet been signed off so will refrain from Approval until then. Everything else looks good to go and happy to approve once that has been signed off.

**Review by janderson2 — APPROVED** [review]

**Pull request merged**
